codegen_glibmm/templates/stub.cpp.templ

```diff
@@ -191,16 +193,19 @@ void {{ class_name_with_namespace }}::{{ signal.name }}_emitter(
     const Glib::VariantContainerBase params =
         Glib::Variant<std::vector<Glib::VariantBase>>::create_tuple(paramsList);
     for (const RegisteredObject &obj: m_registered_objects) {
-        obj.connection->emit_signal(
-            obj.object_path,
-            "{{ signal.iface_name }}",
-            "{{ signal.name }}",
-            Glib::ustring(),
-            params);
+        for (const auto &bus_name: destination_bus_names) {
+            obj.connection->emit_signal(
+                    obj.object_path,
+                    "{{ signal.iface_name }}",
+                    "{{ signal.name }}",
+                    bus_name,
```

Comment:
@mardy 
I try recall and think about that again by my patch was old ):

{} means to size 0 to vector container,
so if "" is added to vector, bus_name = “” and emit_signal works on broadcast signal.
But if vector size is 0 by {}, emit_signal will be never called